### ISSUE TITLE
Upgrade to Bootstrap 3

### ISSUE THREAD

**Issue by nickdos** [AtlasOfLivingAustralia/biocache-hubs#149]
Marie has done this i  Franch portal - https://github.com/melecoq/gbiffrance-hub/tree/master/grails-app/views

so we should be able to use this

**Comment by nickdos** [comment]
conversion utility for html source: http://code.divshot.com/bootstrap3_upgrader/

**Comment by santiagomtzdelariva** [comment]
Yesterday I tried to replicate the gbiffrance-hub, but I got this error: 

There is a dependency on module [bootstrap3] by module [alf] but no such module has been defined.

I don't know where Maire-Elise defines the bootstrap3 module in her generic-hub. :(

I resolved this, adding the pluging of bootstrap in the BuildConfig file: 

runtime ':twitter-bootstrap:3.3.5'

and Also I had to change the name of the dependency by bootstrap.

And now I have the generic-hub-es with bootstrap 3: https://github.com/santiagomtzdelariva/generic-hub-es

I have only included the header and footer with bootstrap, I have to change the other views.
